docs/design/net-topology-aware.md

```diff
@@ -0,0 +1,90 @@
+# Net Topology Aware Plugin
+
+- [Net Topology Aware Plugin](#net-topology-aware-plugin)
```

Comment:
I have a small question: why is this called network topology and not node topology?

Reply:
I see in the design document that all the implementations are related to node labels, and there seems to be no information related to the network.

Reply:
Is it different from the nodegroup plugin?